solutions/swb-reference/src/config/demo.yaml

```diff
@@ -9,9 +9,8 @@ vpcId: ''
 albSubnetIds: []
 ecsSubnetIds: []
 albInternetFacing: true
-# TODO: once this branch in dev, provide appropriate values
-#hostZoneId: ''
-#domainName: ''
+hostedZoneId: 'Z0021810358T3Y7HQSY91'
```

Comment:
Could we get the hostedZoneId from GitHub secrets? Similar to [this](https://github.com/aws-solutions/solution-spark-on-aws/blob/171b97867613bb667220ac4db3986771f1b3d43a/.github/workflows/deploy-integration-swb-reference-byon.yml#L75)